src/TrendsOverview.svelte

```diff
@@ -149,15 +151,21 @@
     );
   }
 
-  function dismissDownloadPopup(event): void {
+  function dismissDownloadPopupOnClick(event): void {
```

Comment:
Nit: I was a little confused since both `dismissDownloadPopup` and `dismissDownloadPopupOnClick` handle click events, and I wondered what the difference was.  `dismissDownloadPopup` closes the box regardless of where the click came from and `dismissDownloadPopupOnClick` only closes it if the click comes outside of the bounding box?

A more descriptive name might help.  Anyway, the rest of the logic looks good.

Reply:
Names changed + added comments.